diffpy/pdfmorph/pdfmorphapp.py

```diff
@@ -62,9 +61,58 @@ def custom_error(self, msg):
     parser.add_option(
         '-s',
         '--save',
-        metavar="FILE",
-        dest="savefile",
-        help="Save manipulated PDF from FILE1 to FILE. Use \'-\' for stdout.",
+        metavar="SLOC",
+        dest="saveloc",
+        help="""Save the manipulated PDF to a file SLOC. Use \'-\' for stdout.
```

Comment:
will a chemist be able to understand this?   I am not sure the target audience (presume a chemist) but please write the help with the target audience in mind.

Reply:
Got it, changed SLOC to NAME.